### PR TITLE
Allows scrolling the viewport using alt + wheel

### PR DESCRIPTION
**Pull request by dhuebner**
To try it out open the SVG example, hold `alt` and use mouse wheel or touchpad to scroll.

See #339

### CODE REVIEW

File: packages/sprotty/src/features/viewport/zoom.ts [hunk 1]
```diff
@@ -51,20 +51,32 @@ export class ZoomMouseListener extends MouseListener {
...
-                    y: viewport.scroll.y - offsetFactor * viewportOffset.y
-                },
-                zoom: viewport.zoom * newZoom
-            };
-            return [SetViewportAction.create(viewport.id, newViewport, { animate: false })];
+        if (!viewport) {
+            return [];
         }
-        return [];
+        const newViewport = () => {
```
Comment (spoenemann) [inline packages/sprotty/src/features/viewport/zoom.ts:57]: Why is this a function? It's executed immediately before the action is created.

**Review by spoenemann — COMMENTED** [review]
Error:
```
src/features/viewport/zoom.ts(18,1): error TS6133: 'Viewport' is declared but its value is never read.
```

File: packages/sprotty/src/features/viewport/zoom.ts (hunk 1 above)
Comment (dhuebner) [reply packages/sprotty/src/features/viewport/zoom.ts:57]: It's not a function, it's a `newViewport` supplier! :)
It looked ugly when using return in if/else branches.  
I will have a closer look soon, maybe extract some code to a protected  method and refactor existing code a bit

Comment (dhuebner) [reply packages/sprotty/src/features/viewport/zoom.ts]: Extracted zoom and scroll viewport calculation in separate functions for easier customization.

**Pull request merged**